### PR TITLE
docs: split and cleanup README

### PR DESCRIPTION
**Pull request by lidel**
This PR aims to improve UX/DX

- README 
  - added Usage and About sections at the top
  - moved longer content to separate docs
  - added missing license info (@laurentsenta please review and sign-off you are ok with dual licensing – [we use it for all other projects](https://protocol.ai/blog/announcing-the-permissive-license-stack/))
  - provisional logo (octopus is https://www.ipdx.co, so created new one for this project)

### CODE REVIEW

File: README.md
Comment (lidel) [inline README.md:1]: :information_source:  the README got very long and this is my attempt at making it more friendly by focusing on "WHAT" it is and "WHY" it exists, and "HOW" being delegated to separate docs

File: docs/web-dashboard.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+### Adding new implementation to the dashboard
+
+The dashboard is hosted at
+[conformance.ipfs.tech](https://conformance.ipfs.tech/). It aggregates test
+outputs from various IPFS implementations and renders them on a static website.
+
+To add a new implementation to the dashboard:
+
+- Ensure that implementation has significant user base and passes conformance tests.
```
Comment (lidel) [inline docs/web-dashboard.md:51]: :information_source:  @laurentsenta  I've added this first point to ensure we only add  "significant" implementations that pass tests. It is very vague on purpose – once we have a real world example of a third-party implementation that wants to be featured, we will adjust the language then.

I suspect nobody will want to implement the entire thing, and we will end up in projects that only show up for `trustless-gateway` etc.

**Review by laurentsenta — APPROVED** [review]
Looks perfect, thanks for taking the time to clear that up and making a new logo :)

**Review by hacdias — APPROVED** [review]
Quite amazing!

**Pull request merged**